gdpr/gdpr.go

```diff
@@ -24,10 +24,21 @@ type Permissions interface {
 	AuctionActivitiesAllowed(ctx context.Context, bidderCoreName openrtb_ext.BidderName, bidder openrtb_ext.BidderName, PublisherID string, gdprSignal Signal, consent string, aliasGVLIDs map[string]uint16) (permissions AuctionPermissions, err error)
 }
 
-type PermissionsBuilder func(config.GDPR, TCF2ConfigReader, map[openrtb_ext.BidderName]uint16, VendorListFetcher) Permissions
+type GVLVendorIDs map[openrtb_ext.BidderName]uint16
 
-// NewPermissions gets an instance of the Permissions for use elsewhere in the project.
-func NewPermissions(cfg config.GDPR, tcf2Config TCF2ConfigReader, vendorIDs map[openrtb_ext.BidderName]uint16, fetcher VendorListFetcher) Permissions {
+type PermissionsBuilderMaker func(config.GDPR, GVLVendorIDs, VendorListFetcher) PermissionsBuilder
```

Comment:
I added it for testability purposes but I don't think we need this right now since the permissions builder is instantiated in the router and I'm not planning to make any adjustments there unless they are necessary. Removed.